Source/Bugsnag/BugsnagNotifier.m

```diff
@@ -72,6 +74,9 @@ void BSSerializeDataCrashHandler(const KSCrashReportWriter *writer) {
     if (g_bugsnag_data.stateJSON) {
         writer->addJSONElement(writer, "state", g_bugsnag_data.stateJSON);
     }
+    if (g_bugsnag_data.onCrash) {
+        g_bugsnag_data.onCrash();
```

Comment:
Can we sensibly pass it anything? The writer? Could we do something sensible with the return value?

Usually we pass the payload in and you can edit it. And then the return value is a bool true/false to send it or not.